python/src/ExperimentIntegration_doc.i.in

```diff
@@ -0,0 +1,163 @@
+%feature("docstring") OT::ExperimentIntegration
+"Create a quadrature rule for numerical integration based on a weighted experiment.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+weightedExperiment : :class:`~openturns.WeightedExperiment`
+    The weighted experimental design.
+
+See also
+--------
+openturns.WeightedExperiment
+
+Notes
+-----
+This class creates an integration method using a weighted experimental
+design.
+It can be used on a function with arbitrary input and
+output.
+
+
+Examples
+--------
+Integrate the Ishigami physical model.
+
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> from openturns.usecases import ishigami_function
+>>> im = ishigami_function.IshigamiModel()
+>>> ot.RandomGenerator.SetSeed(0)
+>>> sampleSize = 32768
+>>> experiment = ot.MonteCarloExperiment(im.distributionX, sampleSize)
+>>> integration = otexp.ExperimentIntegration(experiment)
+>>> approximatedOutputMean = integration.integrate(im.model)
+>>> print(approximatedOutputMean)
+[3.5...]
+
+Compute the L2 norm of a function.
+
+>>> im = ishigami_function.IshigamiModel()
+>>> centeredIshigamiFunction = ot.SymbolicFunction(
+...     ['x1', 'x2', 'x3'],
+...     ['sin(x1) + 7 * (sin(x2)) ^ 2 + 0.1 * x3^4 * sin(x1) - 3.5']
+... )
+>>> ot.RandomGenerator.SetSeed(0)
+>>> sampleSize = 32768
+>>> experiment = ot.MonteCarloExperiment(im.distributionX, sampleSize)
+>>> integration = otexp.ExperimentIntegration(experiment)
+>>> functionNorm = integration.computeL2Norm(centeredIshigamiFunction)
+>>> print(functionNorm)
+[3.7...]
+
+Compute the error between two functions.
+This is easy using the arithmetic operator `-` on two
+instances of the :class:`~openturns.Function` class.
+
+>>> im = ishigami_function.IshigamiModel()
+>>> partOfIshigamiFunction = ot.SymbolicFunction(
+...     ['x1', 'x2', 'x3'], ['7 * (sin(x2)) ^ 2 + 0.1 * x3^4 * sin(x1)']
+... )
+>>> ot.RandomGenerator.SetSeed(0)
+>>> sampleSize = 32768
+>>> experiment = ot.MonteCarloExperiment(im.distributionX, sampleSize)
+>>> integration = otexp.ExperimentIntegration(experiment)
+>>> functionError = integration.computeL2Norm(im.model - partOfIshigamiFunction)
+>>> print(functionError)
+[0.70...]"
```

Comment:
I think this example should be moved to the "Example" section. It does not really present an additional functionality of the class, but is rather a clever combination of this class and of the - operator applied to Functions.

Reply:
Ok. Fixed at 44f577a40